js/whitebit.js

```diff
@@ -974,64 +974,65 @@ module.exports = class whitebit extends Exchange {
          * @param {object} params extra parameters specific to the whitebit api endpoint
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
          */
-        let method = undefined;
         await this.loadMarkets ();
         const market = this.market (symbol);
         const request = {
             'market': market['id'],
             'side': side,
             'amount': this.amountToPrecision (symbol, amount),
         };
-        const stopPrice = this.safeNumber2 (params, 'stopPrice', 'activationPrice');
-        if (stopPrice !== undefined) {
-            // it's a stop order
-            request['activation_price'] = this.priceToPrecision (symbol, stopPrice);
-            if (type === 'limit' || type === 'stopLimit') {
-                // it's a stop-limit-order
-                method = 'v4PrivateOPostOrderStopLimit';
-            } else if (type === 'market' || type === 'stopMarket') {
-                // it's a stop-market-order
-                method = 'v4PrivatePostOrderStopMarket';
+        const isLimitOrder = (type === 'limit') || (type === 'stop_limit');
+        const isMarketOrder = (type === 'market') || (type === 'stop_market');
+        const stopPrice = this.safeNumberN (params, [ 'triggerPrice', 'stopPrice', 'activation_price' ]);
+        const isStopOrder = (stopPrice !== undefined);
+        const timeInForce = this.safeString (params, 'timeInForce');
+        const postOnly = this.isPostOnly (isMarketOrder, false, params);
+        if (postOnly) {
+            throw new NotSupported (this.id + ' createOrder() does not support postOnly orders.');
+        }
+        let method = undefined;
+        if (timeInForce === 'FOK') {
```

Comment:
Since whitebit calls these orders `stock_market`, could we support it?

```Javascript
if (timeInForce === 'FOK' || type === 'stock_market') {
   (...)
}
```